### PR TITLE
Fixing paddings in program settings page and cleaning up html

### PR DESCRIPTION
**Pull request by xinnanlisf**
# Critical Changes

# Changes

# Issues Closed

# New Metadata

## Unpackaged Metadata

# Deleted Metadata

# Testing Notes

### CODE REVIEW

**Review by skosgi1 — APPROVED** [review]
Padding changes and new button alignment fix looks great!

File: force-app/main/default/lwc/programSettings/programSettings.html [hunk 1]
```diff
@@ -42,16 +42,17 @@
...
-            <div class="slds-p-right_x-large slds-text-heading_small slds-p-top_x-large">
+            <div class="slds-col slds-p-right_x-large slds-text-heading_small slds-p-top_x-large">
                 {labelReference.afflCreateFromPrgrmHeading}
                 <lightning-helptext content={labelReference.afflCreateFromPrgrmInfo}></lightning-helptext>
             </div>
             <div class="slds-grid slds-grid_vertical-align-start slds-wrap">
                 <!-- Default Account Record Type Setting -->
                 <c-settings-row-input
-                    class="slds-size_3-of-3 slds-p-top_x-large"
+                    class="slds-col slds-p-top_x-large"
```
Comment (skosgi1) [inline force-app/main/default/lwc/programSettings/programSettings.html:55]:
```suggestion
                    class="slds-col slds-size_1-of-1 slds-p-top_x-large"
```

File: force-app/main/default/lwc/programSettings/programSettings.html [hunk 2]
```diff
@@ -64,7 +65,7 @@
                 ></c-settings-row-input>
 
                 <c-settings-row-input
-                    class="slds-size_3-of-3"
+                    class="slds-col"
```
Comment (skosgi1) [inline force-app/main/default/lwc/programSettings/programSettings.html:68]:
```suggestion
                    class="slds-col slds-size_1-of-1"
```

**Review by skosgi1 — CHANGES_REQUESTED** [review]
couple of minor changes

**Review by skosgi1 — APPROVED** [review]
looks good, approved!

**Comment by rhakeshj** [comment]
Tested this by building a scratch org and the "New" button for auto enrollment mappings under program enrollment is properly aligned now. QA passed

**Pull request merged**